lib/widgets/badge.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:badges/badges.dart';
+import 'package:flutter/material.dart';
+
+class BadgeIcon extends StatelessWidget {
```

Comment:
I don't see this being used anywhere.
Will delete item, add/remove quantity icons be replaced by this widget?

Reply:
It is being used in pos_invoice.dart.